qiskit_acqua/operator.py

```diff
@@ -27,6 +27,7 @@
 from qiskit.wrapper import execute as q_execute
 from qiskit.tools.qi.pauli import Pauli, label_to_pauli, sgn_prod
 from qiskit.qasm import pi
+from copy import deepcopy
```

Comment:
Do you mind moving this import to the above section, which is from the native Python library.